external/fv3fit/tests/emulation/test_data_transforms.py

```diff
@@ -23,17 +23,6 @@ def xr_dataset():
     )
 
 
-def test_open_netcdf(xr_dataset: xr.Dataset):
```

Comment:
I removed this since open_netcdf_dataset is a private function and I added a integration test that covers this case.